### PR TITLE
Add link to ars article

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change

Add link to arstechnica.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/index.html [hunk 1]
```diff
@@ -223,6 +223,12 @@ <h1>Home Assistant is sponsored by</h1>
       <div class="material-card">
         <h1>Home Assistant in the press</h1>
 
+        <a href="https://arstechnica.com/information-technology/2021/03/how-to-achieve-smart-home-nirvana-or-home-automation-without-subscription/"
```
Comment (ludeeus) [inline source/index.html:226]: This should target _blank, and maybe also use noreferrer?

Comment (balloob) [reply source/index.html:226]: I've followed the same pattern as all the others. Why would we use noreferrer ? I understand noopener, but it's fine for them to know we link to them?

Comment (ludeeus) [reply source/index.html:226]: I always mix them.... noopener is what I meant

Comment (balloob) [reply source/index.html:226]: btw, 
image

**Review by ludeeus — APPROVED** [review]

**Pull request merged**